<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Courier 10 Pitch</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Cute Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit" >
      <property name="font" >
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0" >
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="lineWrapMode" >
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="html" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Courier 10 Pitch'; font-size:11pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRepeat" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="windowTitle" >
     <string>Cute Editor</string>
    </property>
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionInfo" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Edit" />
   <addaction name="menuSettings" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="cursor" >
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="acceptDrops" >
    <bool>false</bool>
   </property>
   <property name="windowTitle" >
    <string>Main toolbar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="allowedAreas" >
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionCut" />
   <addaction name="actionCopy" />
   <addaction name="actionPaste" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/file/icons/document-new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="toolTip" >
    <string>New Document</string>
   </property>
   <property name="whatsThis" >
    <string>Makes a new document</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/file/icons/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip" >
    <string>Open Document</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/file/icons/media-floppy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="toolTip" >
    <string>Save Document</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip" >
    <string>Save As...</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/file/icons/emblem-unreadable.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip" >
    <string>Quit Application</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/edit/icons/edit-undo.png</iconset>
   </property>
   <property name="text" >
    <string>Undo</string>
   </property>
   <property name="toolTip" >
    <string>Undo Type</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRepeat" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/edit/icons/edit-redo.png</iconset>
   </property>
   <property name="text" >
    <string>Repeat</string>
   </property>
   <property name="toolTip" >
    <string>Repeat Type</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/edit/icons/edit-cut.png</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="toolTip" >
    <string>Cut</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/edit/icons/edit-copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/edit/icons/edit-paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionInfo" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/toolbar/file/icons/battery.png</iconset>
   </property>
   <property name="text" >
    <string>Info...</string>
   </property>
   <property name="toolTip" >
    <string>Info...</string>
   </property>
   <property name="font" >
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
